### PR TITLE
update packages and install ntp 

### PR DESCRIPTION
**Pull request by ankul-shippable**
#824

### CODE REVIEW

File: scripts/remote/installNTP.sh [hunk 1]
```diff
@@ -1,10 +1,15 @@
 #!/bin/bash -e
 
 ntp_install() {
-  echo "Installing and Starting NTP"
-
-  apt-get install -y ntp
-  /etc/init.d/ntp restart
+  ntp_status_res=$(service ntp status)
+  if [ -z "$ntp_status_res" ]; then
```
Comment (mohit5it9) [inline scripts/remote/installNTP.sh]: shouldn't this be ntp_status_res=$(service ntp status | grep "unrecognized")

Comment (ankul-shippable) [inline scripts/remote/installNTP.sh]: this is to check whether ntp is installed or not, it doesn't return anything if its not installed

Comment (mohit5it9) [inline scripts/remote/installNTP.sh]:
```
service ntp status
ntp: unrecognized service
```

File: scripts/installCore.sh [hunk 2]
```diff
@@ -21,22 +21,25 @@ _check_component_status() {
...
+    _exec_remote_cmd "$host" "$SCRIPT_DIR_REMOTE/installBase.sh $INSTALL_MODE"
   done
+  UPDATED_APT_PACKAGES=true
 }
 
 install_docker() {
   SKIP_STEP=false
   _check_component_status "dockerInstalled"
   if [ "$SKIP_STEP" == false ]; then
+    if [ "$UPDATED_APT_PACKAGES" == false ]; then
```
Comment (shrivara) [inline scripts/installCore.sh]: Let's move this to the `bootstrap` method so we don't have to do it again. We can rename the method from `bootstrap` to `ensure_updated_packages` to make it clearer what we're trying to do here. The rest lgtm.

**Pull request merged**